ui/client/containers/NussknackerApp.tsx

```diff
@@ -25,10 +24,28 @@ import ErrorHandler from "./ErrorHandler"
 import {ProcessTabs} from "./ProcessTabs"
 import {getFeatureSettings} from "../reducers/selectors/settings"
 import CustomTabs from "./CustomTabs"
+import {withTranslation} from "react-i18next"
+import {WithTranslation} from "react-i18next/src"
+import {compose} from "redux"
+import {UnregisterCallback} from "history"
+import ProcessBackButton from "../components/Process/ProcessBackButton"
 
-export class EspApp extends React.Component {
+type OwnProps = {}
+type State = {}
+
+type MetricParam = {
+  params: {
+    processId: string,
+  },
+}
+
+class NussknackerApp extends React.Component<Props, State> {
+
+  private readonly mountedHistory: UnregisterCallback
+
+  constructor(props) {
+    super(props)
```

Comment:
is componentDidMount the same as constructor in our case?

Reply:
It works as same as before .. So I guess it's ok. I've tested app without this assign and application did not crash..